sources/packages/backend/libs/services/src/constants/assessment.constants.ts

```diff
@@ -0,0 +1 @@
+export const MASKED_MSFAA_NUMBER = "XXXXXXXXXX";
```

Comment:
Was wondering why it is moved to lib as it is is not used anywhere outside API. Is it for an upcoming PR?